core/src/toga/handlers.py

```diff
@@ -262,3 +263,25 @@ def __bool__(self, other: object) -> NoReturn:
 
 class PermissionResult(AsyncResult):
     RESULT_TYPE = "permission"
+
+
+class WeakrefCallable:
+    """
+    A wrapper for callable that holds a weak reference to it.
+
+    This can be useful in particular when setting winforms event handlers, to avoid
```

Comment:
```suggestion
    This can be useful in particular when setting WinForms event handlers, to avoid
```
While we're tweaking WeakrefCallable, might as well fix the case for WinForms.

[Sidenote: iOS Testbed failure is unrelated.